core/wiki/peek-stylesheets.tid

```diff
@@ -1,76 +1,52 @@
-<$wikify name="styles" text=<<source>>>
+\procedure expandable-content()
+<$wikify name="styles" text={{!!text}}>
 <$codeblock code=<<styles>> language="css"/>
 </$wikify>
-</$set>
-</$reveal>
-</li>
-</$set>
-</$vars>
-</$list>
-</ol>
 \end
 
-\define stylesheets-list()
+\procedure toggle-content()
+<$parameters openState="" open="[<openState>substitute[]get[text]else[no]]" openImage="$:/core/images/down-arrow" closedImage="$:/core/images/right-arrow" openCaption="" closedCaption="">
+<%if [<openState>!is[blank]] %>
+<$checkbox tiddler={{{ [<openState>substitute[]] }}} field="text" checked="yes" style.appearance="none">
+<span style.cursor="pointer">
+<%if [subfilter<open>match[yes]] %>
+<$transclude $tiddler=<<openImage>> size="1em" /> <<openCaption>>
+<%else%>
+<$transclude $tiddler=<<closedImage>> size="1em" /> <<closedCaption>>
+<%endif%>
+</span>
+</$checkbox>
+<%endif%>
+</$parameters>
+\end
+
+\procedure expandable-list()
 \whitespace trim
+<$parameters filter="" openState="" open="[<openState>substitute[]get[text]else[no]]">
 <ol>
-<$list filter="[all[shadows+tiddlers]tag[$:/tags/Stylesheet]!has[draft.of]]">
+<$list filter=<<filter>> >
 <li>
-<$link>
-<$view field="title"/>
-</$link>
-<$set name="source" tiddler=<<currentTiddler>>>
-<$wikify name="styles" text=<<source>>>
-<pre>
-<code>
-<$text text=<<styles>>/>
-</code>
-</pre>
-</$wikify>
-</$set>
+<$transclude $variable="toggle-content" open=<<open>> openState=<<openState>> />
+<$link/>
+<%if [subfilter<open>match[yes]]%>
+<<expandable-content>>
+<%endif%>
 </li>
 </$list>
 </ol>
+</$parameters>
 \end
-\whitespace trim
 
-<$vars modeState=<<qualify "$:/state/peek-stylesheets/mode/">>>
+<$let modeState=`$:/state/peek-stylesheets/mode/$(qualify)$` mode={{{ [<modeState>get[text]else[expand]] }}} listFilter="[all[shadows+tiddlers]tag[$:/tags/Stylesheet]!has[draft.of]]">
 
-<$reveal type="nomatch" state=<<modeState>> text="expanded" tag="div">
-<$button set=<<modeState>> setTo="expanded" class="tc-btn-invisible">{{$:/core/images/chevron-right}} {{$:/language/ControlPanel/Stylesheets/Expand/Caption}}</$button>
-</$reveal>
-<$reveal type="match" state=<<modeState>> text="expanded" tag="div">
-<$button set=<<modeState>> setTo="restored" class="tc-btn-invisible">{{$:/core/images/chevron-down}} {{$:/language/ControlPanel/Stylesheets/Restore/Caption}}</$button>
-</$reveal>
+<$transclude $variable="toggle-content" openImage="$:/core/images/chevron-right" closedImage="$:/core/images/chevron-down" openState=<<modeState>> openCaption={{$:/language/ControlPanel/Stylesheets/Expand/Caption}} closedCaption={{$:/language/ControlPanel/Stylesheets/Restore/Caption}}/>
 
-<$reveal type="nomatch" state=<<modeState>> text="expanded" tag="div">
-<<expandable-stylesheets-list>>
-</$reveal>
-<$reveal type="match" state=<<modeState>> text="expanded" tag="div">
-<<stylesheets-list>>
-</$reveal>
+<%if [<mode>match[expand]]%>
+<$transclude $variable="expandable-list" filter=<<listFilter>> open="yes"/>
```

Comment:
The "open", "closed" state value is wrong. It is `expanded` and `restored` 
Have a look at the state: https://tiddlywiki.com/#%24%3A%2Fstate%2Fpeek-stylesheets%2Fmode%2F-1116772087 at tiddlywiki.com and at: https://deploy-preview-8794--tiddlywiki-previews.netlify.app/#%24%3A%2Fstate%2Fpeek-stylesheets%2Fmode%2F-1116772087

**current Version**

![image](https://github.com/user-attachments/assets/1ddd7529-e163-459a-bf5a-4a4bfdcdcb7f)

**PR version**

![image](https://github.com/user-attachments/assets/1b944fa8-12b7-41e8-b44f-5d4d2ae642bc)

It is not possible to change those values for backwards compatibility reasons. There may be unwanted side effects.

Reply:
Understood, sorry for the oversight 😅 I'm marking this PR as a draft for now and will request a review after more thorough testing.